### PR TITLE
Correct static errors and source code comments.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@nixpanic @Madhu-1 ptal.

**Comment by humblec** [comment]
> Changes look good. But the text in the PR is pretty useless, please clean that up as it cause only confusion and ugliness.

@nixpanic can you suggest if you have anything specific in mind. I can accommodate the same.

File: pkg/util/conn_pool.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ type connEntry struct {
 	users    int
 }
 
+//ConnPool is the struct which contains details of connection entries in the pool and gc controlled params.
```
Comment (phlogistonjohn) [inline pkg/util/conn_pool.go]: Minor nit: typical style is to have a space between the `//` and the word.

File: pkg/util/conn_pool.go [hunk 2]
```diff
@@ -45,7 +46,7 @@ type ConnPool struct {
 	conns map[string]*connEntry
 }
 
-// Create a new ConnPool instance and start the garbage collector running
+// NewConnPool create a new connection pool instance and start the garbage collector running
```
Comment (phlogistonjohn) [inline pkg/util/conn_pool.go]: Grammar nit: "creates" not "create", "starts" not "start"

File: pkg/util/conn_pool.go [hunk 3]
```diff
@@ -76,7 +77,7 @@ func (cp *ConnPool) gc() {
 	cp.timer.Reset(cp.interval)
 }
 
-// Stop the garbage collector and destroy all connections in the pool.
+// Destroy stop the garbage collector and destroy all connections in the pool.
```
Comment (phlogistonjohn) [inline pkg/util/conn_pool.go]: Grammar: "stops" not "stop", "destroy" becomes "destroys"

File: pkg/util/conn_pool.go [hunk 4]
```diff
@@ -116,7 +117,7 @@ func (cp *ConnPool) getConn(unique string) *rados.Conn {
 	return nil
 }
 
-// Return a rados.Conn for the given arguments. Creates a new rados.Conn in
+// Get return a rados.Conn for the given arguments. Creates a new rados.Conn in
```
Comment (phlogistonjohn) [inline pkg/util/conn_pool.go]: "Get returns"

File: pkg/util/conn_pool.go [hunk 5]
```diff
@@ -168,7 +169,7 @@ func (cp *ConnPool) Get(pool, monitors, keyfile string) (*rados.Conn, error) {
 	return conn, nil
 }
 
-// Reduce the reference count of the rados.Conn object that was returned with
+// Put reduce the reference count of the rados.Conn object that was returned with
```
Comment (phlogistonjohn) [inline pkg/util/conn_pool.go]: "Put reduces"

File: pkg/cephfs/controllerserver.go [hunk 6]
```diff
@@ -311,7 +310,7 @@ func (cs *ControllerServer) ValidateVolumeCapabilities(
 	}, nil
 }
 
-// ExpandVolume expand CephFS Volumes on demand based on resizer request
+// ControllerExpandVolume expand CephFS Volumes on demand based on resizer request
```
Comment (phlogistonjohn) [inline pkg/cephfs/controllerserver.go]: Grammar nit: when we're treating the first word as a noun we want the verb to agree, so "expand" needs to become "expands"

**Comment by humblec** [comment]
@phlogistonjohn @nixpanic ptal.

File: pkg/util/conn_pool.go (hunk 2 above)
Comment (phlogistonjohn) [reply pkg/util/conn_pool.go]: Unless github is caching something it still looks like "create" to me. Should be "creates".

**Comment by nixpanic** [comment]
On Thu, Mar 26, 2020 at 06:38:13AM -0700, Humble Devassy Chirammal wrote:
> > Changes look good. But the text in the PR is pretty useless, please clean that up as it cause only confusion and ugliness.
> 
> @nixpanic can you suggest if you have anything specific in mind. I can accommodate the same.

At least delete the template text and describe what tool you used to
check it (ideally with cmdline arguments or config file).

**Comment by humblec** [comment]
@nixpanic removed the unwanted description.

**Comment by humblec** [comment]
> Strange, I did leave a comment earlier with a better explanation for the PR comment text. Not sure where it went or why it was not saved....
> 
> I asked what tool was used for this, and provide the commandline arguments and/or configuration. We should add it as a CI job so that we do not regress in the future.

@nixpanic it happened in IDE, I think they have `golang-ci` or `linter` enabled. May be our CI does not enable the linter/static check .

**Pull request merged**